src/compressed_tensors/config/base.py

```diff
@@ -37,11 +37,16 @@ class SparsityCompressionConfig(RegistryMixin, BaseModel):
     Base data class for storing sparsity compression parameters
 
     :param format: name of compression format
+    :param targets: List of layer names or layer types that aren't sparse and should
+        be ignored during compression. By default, assume all layers are targeted
+    :param ignore: List of layer names (unique) to ignore from targets. Defaults to None
```

Comment:
would make some comment about these supporting regex

Reply:
They don't as of now; I'll update the docstring when and if that functionality is added!